deploy/role.yaml

```diff
@@ -13,12 +13,6 @@ rules:
   - delete
   - update
   - list
-- apiGroups:
-  - apiextensions.k8s.io
-  resources:
-  - customresourcedefinitions
-  verbs:
-  - get
```

Comment:
Can you refresh my memory here, why is this in the rbac in the first place and why is it being removed now?

Reply:
[Previously](https://github.com/operator-framework/operator-marketplace/pull/134/files#diff-a5666326aefdf43a8e89fd4871449fa0L62) the status package checked that the ClusterOperator CRD existed. Now we use the `k8sinterface.Discovery` approach.